build/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY . .
 RUN make build
 
 ####
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.10-1086
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.4-1227.1726694542
```

Comment:
Should be  `registry.access.redhat.com/ubi9-minimal:9.4-1227.1726694542`.  See https://catalog.redhat.com/software/containers/ubi9-minimal/61832888c0d15aff4912fe0d?container-tabs=gti&gti-tabs=unauthenticated

Reply:
This works too, https://catalog.redhat.com/software/containers/ubi9/ubi-minimal/615bd9b4075b022acc111bf5?container-tabs=gti